### PR TITLE
Support MySQL and add test to verify database connection

### LINKED ISSUE #33: Support MySQL and add tests to verify database connection works
Test to ensure MySQL connection works.

**Requirements**
- Add MySQL ODBC driver
- Add test for MySQL connection
- Integrate it in Travis

Take reference from PR https://github.com/laudio/pyodbc/pull/1 on how it's done for Postgres and MSSQL; need to add MySQL in that.

Review existing tests here https://github.com/laudio/pyodbc/tree/master/test

### PR DESCRIPTION
**Pull request by parwatcodes**
Resolves #33 

- Test case for MySQL connection.
- Base docker image contains mysql odbc driver.
- Travis runs mysql service with the test credentials provided.
- Image size is now 290 MB 
   - `bin/` is copied which is 10.5 MB
   - `lib/` is copied which is 50 MB

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -9,3 +9,9 @@ TEST_PG_DB_HOST=0.0.0.0
 TEST_PG_DB_NAME=testdb
 TEST_PG_DB_USER=testdb
 TEST_PG_DB_PASSWORD=TestPassword@1234
+
+#Database connection for MySQL
```
Comment (mesaugat) [inline .env.example]:
```suggestion
# Database connection for MySQL
```

File: test/test_connection.py [hunk 2]
```diff
@@ -1,17 +1,24 @@
...
-    result = exec_query(PG, "SELECT 'It works!'")
+    result = exec_query(MSSQL, "SELECT 'It works!'")
 
     assert result == 'It works!'
 
 
 def test_pg():
     ''' Test connection to PostgreSQL Server. '''
-    result = exec_query(MSSQL, "SELECT 'It works!'")
+    result = exec_query(PG, "SELECT 'It works!'")
```
Comment (kabirbaidhya) [inline test/test_connection.py:15]: Good catch :+1:

**Comment by kabirbaidhya** [comment]
@p0k8h Next steps:
 1. Update the base docker image to add in the mysql odbc connector.
 2. Update travis config to run mysql server with the test creds you've defined for mysql.
 3. Check the image size diff - prev vs now and optimize the docker image if needed.
 4. Ensure the CI build passes; if no - repeat.

File: Dockerfile [hunk 3]
```diff
@@ -5,16 +5,24 @@ FROM python:3.7-slim AS base
 
 COPY requirements.txt .
 
+ARG FILE_NAME='mysql-connector-odbc-8.0.18-linux-glibc2.12-x86-64bit'
```
Comment (mesaugat) [inline Dockerfile]: Rename to `MYSQL_CONNECTOR`

**Review by kabirbaidhya — APPROVED** [review]

**Review by kabirbaidhya — APPROVED** [review]

**Pull request merged**